### PR TITLE
Update sql-server-backup-to-url-best-practices-and-troubleshooting.md

### PR DESCRIPTION
**Pull request by ujpat**
ICM https://portal.microsofticm.com/imp/v3/incidents/incident/424962517/summary

### CODE REVIEW

**Comment by ujpat** [comment]
@bluefooted for help with review

**Comment by ShannonLeavitt** [comment]
@WilliamDAssafMSFT

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/relational-databases/backup-restore/sql-server-backup-to-url-best-practices-and-troubleshooting.md [hunk 1]
```diff
@@ -214,6 +214,8 @@ To resolve this issue, create a configuration file that allows the Backup to URL
 
 1. Place the configuration file in the Binn folder of the [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] Instance. For example, if my [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] is installed on the C drive of the machine, place the configuration file in `C:\Program Files\Microsoft SQL Server\MSSQL13.\<InstanceName>\MSSQL\Binn`.
 
+1. BackuptoURL.exe is not called when using SAS keys but is triggered when using Access key. Make sure you are using Access keys or you may recieve an error "Operating system error 50(The request is not supported.)"
```
Comment (bluefooted) [inline docs/relational-databases/backup-restore/sql-server-backup-to-url-best-practices-and-troubleshooting.md]:
```suggestion
1. `BackuptoURL.exe` is not called when using [SAS keys](/azure/storage/blobs/storage-blob-user-delegation-sas-create-powershell) but is triggered when using an [access key](/azure/storage/common/storage-account-keys-manage?tabs=azure-portal). Make sure you are using access keys or you may receive an error "Operating system error 50(The request is not supported.)"
```

**Review by bluefooted — CHANGES_REQUESTED** [review]

**Comment by ujpat** [comment]
#sign-off

**Comment by rwestMSFT** [comment]
@ujpat If you want to have sign-off rights on your own PRs, you need to open them in the private `sql-docs-pr` repository.

@bluefooted Can we sign this off?

File: docs/relational-databases/backup-restore/sql-server-backup-to-url-best-practices-and-troubleshooting.md [hunk 2]
```diff
@@ -214,6 +214,8 @@ To resolve this issue, create a configuration file that allows the Backup to URL
 
 1. Place the configuration file in the Binn folder of the [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] Instance. For example, if my [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] is installed on the C drive of the machine, place the configuration file in `C:\Program Files\Microsoft SQL Server\MSSQL13.\<InstanceName>\MSSQL\Binn`.
 
+1. `BackuptoURL.exe` is not called when using [SAS keys](/azure/storage/blobs/storage-blob-user-delegation-sas-create-powershell) but is triggered when using an [access key](/azure/storage/common/storage-account-keys-manage?tabs=azure-portal). Make sure you are using access keys or you may receive an error "Operating system error 50(The request is not supported.)"
```
Comment (rwestMSFT) [inline docs/relational-databases/backup-restore/sql-server-backup-to-url-best-practices-and-troubleshooting.md]:
```suggestion
1. `BackuptoURL.exe` isn't called when using [SAS keys](/azure/storage/blobs/storage-blob-user-delegation-sas-create-powershell), but is triggered when using an [access key](/azure/storage/common/storage-account-keys-manage?tabs=azure-portal). Make sure you use access keys, or you may receive the following error:

> Operating system error 50(The request is not supported.)
```

**Comment by rwestMSFT** [comment]
Pulling this into an internal PR because of the "build pending" issue.

**Pull request closed without merging**